exercise/curry2.js

```diff
@@ -0,0 +1,44 @@
+/*
+## Curry 2
+
+### Instructions
+
+create the following functions with the "currying" process:
+
+-Mult2 that mutiplies two numbers.
+-Add3 that adds three numbers.
+-Sub4 that substract four numbers.
```

Comment:
please format your markdown properly.
note that you can pass prettier on markdown.